_layouts/post.html

```diff
@@ -0,0 +1,111 @@
+---
+layout: default
+---
+<!-- PAGE TOP -->
+<section class="page-title">
+  <div class="container">
+
+    <header>
+
+      <ul class="breadcrumb"><!-- breadcrumb -->
+        <li><a href="/blog/index.html">Blog</a></li>
+        <li class="active"></li>
+      </ul><!-- /breadcrumb -->
+
+    </header>
+
+  </div>      
+</section>
+<!-- /PAGE TOP -->
+
+<!-- CONTENT -->
+<section>
+  <div class="container">
+    <div id="blog" class="row">
+
+      <!-- BLOG ARTICLE -->
+      <div class="col-md-9 col-sm-9">
+
+        <div class="prev-article row">
+
+          <div class="blog-prev-date col-md-3 col-sm-3 hidden-xs" style="margin-top: 20px;">
+            <span class="date">
+              {{page.date | date: "%d"}} <small>{{page.date | date: "%B %Y"}}</small>
+            </span>
+            {% for author in site.data.our-team %}
```

Comment:
My guess is this won't be working for the codebridge site. You'll need to copy over the _data/our-team.json file and the team photos for this to work.

Alternatively just remove this for now.